bigtable/beam/workload-generator/src/test/java/bigtable/WorkloadGeneratorTest.java

```diff
@@ -86,11 +82,18 @@ private static String requireEnv(String varName) {
   public static void beforeClass() {
     projectId = requireEnv("GOOGLE_CLOUD_PROJECT");
     instanceId = requireEnv("BIGTABLE_TESTING_INSTANCE");
-    try (Connection connection = BigtableConfiguration.connect(projectId, instanceId)) {
-      Admin admin = connection.getAdmin();
-      HTableDescriptor descriptor = new HTableDescriptor(TableName.valueOf(TABLE_ID));
-      descriptor.addFamily(new HColumnDescriptor(COLUMN_FAMILY_NAME));
-      admin.createTable(descriptor);
+
+    try {
+      BigtableTableAdminSettings adminSettings =
+          BigtableTableAdminSettings.newBuilder()
+              .setProjectId(projectId)
+              .setInstanceId(instanceId)
+              .build();
+
+      adminClient = BigtableTableAdminClient.create(adminSettings);
+      CreateTableRequest createTableRequest =
+          CreateTableRequest.of(TABLE_ID).addFamily(COLUMN_FAMILY_NAME);
+      adminClient.createTable(createTableRequest);
     } catch (Exception e) {
       System.out.println("Error during beforeClass: \n" + e.toString());
```

Comment:
why is this swallowing exceptions? Does it make sense for the test to continue if the table cant be created?